### PR TITLE
Fix Lutris profile

### PR DESCRIPTION
**Pull request by RandomVoid**
Recently I updated my OS to Ubuntu 21.04 and I discovered that profile for Lutris has been added. Unfortunately Lutris v0.5.8.3 didn't start when I tried to use it. After spending some time on debugging I found two problems (both are fixed in this pull request):
1. Lutris is using ldconfig, so it requires access to /sbin and /usr/sbin. I've added noblacklist option for both paths.
2. Games with enabled Feral GameMode won't run if D-Bus is disabled. I've disabled options dbus-user and dbus-system.

### CODE REVIEW

File: etc/profile-a-l/lutris.profile [hunk 1]
```diff
@@ -70,5 +74,6 @@ shell none
 #private-dev
 private-tmp
 
-dbus-user none
-dbus-system none
+# Games with enabled Feral GameMode won't run if D-Bus system is disabled.
+#dbus-user none
+#dbus-system none
```
Comment (rusty-snake) [inline etc/profile-a-l/lutris.profile:79]: Does `dbus-system none` really break gamemode?

File: etc/profile-a-l/lutris.profile [hunk 2]
```diff
@@ -70,5 +74,6 @@ shell none
 #private-dev
 private-tmp
 
-dbus-user none
-dbus-system none
+# Games with enabled Feral GameMode won't run if D-Bus system is disabled.
+#dbus-user none
```
Comment (rusty-snake) [inline etc/profile-a-l/lutris.profile:78]: Does this work?
```suggestion
dbus-user filter
dbus-user.talk com.feralinteractive.GameMode
```

**Review by rusty-snake — COMMENTED** [review]
LGTM, but I think we can keep some of the dbus restrictions.

File: etc/profile-a-l/lutris.profile (hunk 1 above)
Comment (RandomVoid) [reply etc/profile-a-l/lutris.profile:79]: You're right. `dbus-system none` doesn't break GameMode.

**Comment by rusty-snake** [comment]
FWIW: I did some playing with `gamemoderun` in #4201.

File: etc/profile-a-l/lutris.profile (hunk 2 above)
Comment (RandomVoid) [reply etc/profile-a-l/lutris.profile:78]: Unfortunately Lutris won't start with this options. Following message is logged:
```
(lutris:5): dbind-WARNING **: 18:17:44.334: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Gky7oRM3c7: No such file or directory
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Comment (rusty-snake) [reply etc/profile-a-l/lutris.profile:78]: Try this, if lutris still don't work we need to go with `#dbus-user none`.
```
dbus-user filter
dbus-user.own net.lutris.Lutris
dbus-user.talk com.feralinteractive.GameMode
#dbus-user.talk com.deepin.WMSwitcher
#dbus-user.talk org.gnome.Mutter.DisplayConfig
#dbus-user.talk org.kde.KWin
#dbus-user.talk org.freedesktop.ScreenSaver
#dbus-user.talk org.mate.ScreenSaver
#dbus-user.talk org.xfce.ScreenSaver
```

File: etc/profile-a-l/lutris.profile (hunk 1 above)
Comment (RandomVoid) [reply etc/profile-a-l/lutris.profile]: I've updated commit.

File: etc/profile-a-l/lutris.profile (hunk 2 above)
Comment (RandomVoid) [reply etc/profile-a-l/lutris.profile]: Thanks for suggestion. Following configuration works correctly:
```
dbus-user filter
dbus-user.own net.lutris.Lutris
dbus-user.talk com.feralinteractive.GameMode
```
I've updated commit.

File: etc/profile-a-l/lutris.profile [hunk 3]
```diff
@@ -70,5 +74,8 @@ shell none
 #private-dev
 private-tmp
 
-dbus-user none
+# Games with enabled Feral GameMode won't run if D-Bus user is none.
```
Comment (rusty-snake) [inline etc/profile-a-l/lutris.profile:77]: IMHO no need
```suggestion
```

**Review by rusty-snake — APPROVED** [review]

File: etc/profile-a-l/lutris.profile (hunk 3 above)
Comment (RandomVoid) [reply etc/profile-a-l/lutris.profile]: OK, commit updated.

**Comment by netblue30** [comment]
all in, thanks!

**Pull request merged**